.openshift-interop/README.md

```diff
@@ -0,0 +1,67 @@
+# Openshift CI
+
+This folder contains files, required for OpenShift CI.
+It allows us to verify, that Quarkus works on new version of Openshift.
+
+## How it works
+
+The tests are run regularly (every Monday and the time of writing). Test execution is based on configs: [[1]](https://github.com/openshift/release),[[2]](https://github.com/openshift/release/pull/40279/files).
+
+During the run CI builds an image from the Dockerfile in this folder, and runs some tests from the TS.
+
+Results are posted into [quarkus-qe](https://redhat-internal.slack.com/archives/C05CMLUAWTT) channel in Slack and new issue in [QQE Jira](https://issues.redhat.com/projects/QQE/summary) is created automatically for every failure. Job history can be accessed from the [Prow Dashboard](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-quarkus-qe-quarkus-test-suite-main-quarkus-ocp4.14-lp-interop-quarkus-interop-aws) or via [dashboard](https://testgrid.k8s.io/redhat-openshift-lp-interop-release-4.14-informing#periodic-ci-quarkus-qe-quarkus-test-suite-main-quarkus-ocp4.14-lp-interop-quarkus-interop-aws&width=90).
+
+## Requirements
+### Released version of Quarkus
+Due to the nature of tests, only the released bits are used. These are pulled directly from the maven.repository.redhat.com, not from an internal source, to be as close to real customer use-case as possible. This means that new versions of Quarkus can only be tested once the release is done and the artifacts are in the maven repository.
+
+### OpenShift requirements
+There are no specific requirement on OpenShift installation, but pull secret for dockerhub is highly recommended to avoid pull rate limiting.
+
+### Test machine requirements
+All the required libraries are installed as part of the scenario:
+- Git
+- Java 11 OpenJDK
+- Maven
+- Docker/Podman/Buildah
+
+## Processes
+### Contacting PIT team
+- Email: [pit-qe@redhat.com](mailto:pit-qe@redhat.com)
+- Jira: [INTEROP project](https://projects.engineering.redhat.com/browse/INTEROP)
+- Slack: [#forum-qe-layered-product](https://redhat-internal.slack.com/archives/C04QDE5TK1C)
```

Comment:
`engineering.redhat.com` is behind VPN and the Slack is employee only. Maybe the contact should be in signpost rather than here? Not sure we should be exposing product initiatives in GH.

Reply:
Actually I am kinda split on that, cos splitting docs is never too useful. 

Resolving.